src/test/java/uk/gov/ons/ctp/integration/censusfieldsvc/service/impl/LauncherServiceImplTest.java

```diff
@@ -36,6 +37,182 @@
   private static final String A_URL_RESULT =
       "https://" + A_HOST + "/session?token=" + A_DUMMY_ENCRYPTED_PAYLOAD;
 
+  private static final String SIGNING_PUBLIC_SHA1 = "f008897e548d5c4fb9271e5a85f6fc4759301f26";
```

Comment:
You have overcomplicated this.
This is not needed since the details of the keystore is unimportant to these tests.
Replace with 
```
@Mock private KeyStore keyStoreEncryption;
```